zerver/lib/event_types.py

```diff
@@ -283,6 +283,20 @@ class EventOnboardingSteps(BaseEvent):
     onboarding_steps: list[OnboardingSteps]
 
 
+class EventPushAccountUpdate(BaseEvent):
+    type: Literal["push_account"]
+    op: Literal["update"]
+    push_account_id: str
+    status: Literal["active"]
+
+
+class EventPushAccountRemove(BaseEvent):
+    type: Literal["push_account"]
+    op: Literal["remove"]
+    push_account_id: str
+    reason: Literal["Invalid bouncer_public_key"] | None
```

Comment:
`reason: str` seems like a better type here -- other error strings seem like they should be possible.

And should `reason` be required?

Reply:
As per our recent discussion, there'd be no event with `"op: remove"`. Only `op: "update"`, and `error_code` (instead of reason) when status="failed".